net-analyzer/iplog/iplog-2.2.3-r3.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="TCP/IP traffic logger"
+HOMEPAGE="http://ojnk.sourceforge.net/"
+SRC_URI="mirror://sourceforge/ojnk/${P}.tar.gz"
+
+LICENSE="|| ( GPL-2 FDL-1.1 )"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~mips ~ppc ~sparc ~x86"
+
+DEPEND="net-libs/libpcap"
+RDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}"/${PV}-DLT_LINUX_SSL.patch )
+
+DOCS=( AUTHORS NEWS README TODO example-iplog.conf )
+
+src_compile() {
+	emake CC="$(tc-getCC)" CFLAGS="${CFLAGS} -D_REENTRANT" all
```

Comment:
Defines like this (-DFOO) belong in CPPFLAGS if the build system respects it. Can you try append-cppflags? You may have to pass in CPPFLAGS like you are now for CFLAGS.

Reply:
Updated. As you suggested i had to pass `CPPFLAGS` so that it works.